packages/matrix/docker/synapse/index.ts

```diff
@@ -366,6 +368,7 @@ export async function updateUser(
             })),
           }
         : {}),
+      ...(stripeCustomerId ? { stripe_customer_id: stripeCustomerId } : {}),
```

Comment:
this is not matix API, this is synapse API. I don't think this API supports arbitrary values being set. and moreover, I thought we were relating the stripe ID to a user in our DB not in matrix

Reply:
This is the documentation for this synapse API https://matrix-org.github.io/synapse/latest/admin_api/user_admin_api.html#create-or-modify-account